src/index.html

```diff
@@ -5,8 +5,280 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com/%22%3E">
+    <link rel="preconnect" href="https://fonts.gstatic.com/%22%3E">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="home">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          class="logo__image"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="cards">
+        <div class="card cards__container" data-qa="card">
+          <img
+            class="card__logo"
+            src="images/imac.jpeg"
+            alt="tovar_logo"
+          >
+          <div class="card_info">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+          <div class="card_kod">Код товара: 195434</div>
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+            </div>
+            <div class="card__review-col">Отзывов: 5</div>
+          </div>
```

Comment:
Check the 1st point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md). Use `section` tag, to group elements